### PR TITLE
Enable more documentation warnings

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Enable more documentation warnings

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2721)

### CODE REVIEW

File: src/System.Windows.Forms.Primitives/src/Interop/ComCtl32/Interop.MCHITTESTINFO.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ internal static partial class Interop
     internal static partial class ComCtl32
     {
         /// <summary>
-        /// <see cref="https://docs.microsoft.com/en-us/windows/win32/api/commctrl/ns-commctrl-mchittestinfo"/>
+        /// See https://docs.microsoft.com/en-us/windows/win32/api/commctrl/ns-commctrl-mchittestinfo
```
Comment (sharwell) [inline src/System.Windows.Forms.Primitives/src/Interop/ComCtl32/Interop.MCHITTESTINFO.cs]:
```
<see href="https://docs.microsoft.com/en-us/windows/win32/api/commctrl/ns-commctrl-mchittestinfo">MCHITTESTINFO structure (Microsoft Docs)</see>
```

File: src/System.Windows.Forms.Primitives/src/Interop/Ole32/Interop.IServiceProvider.cs [hunk 2]
```diff
@@ -17,7 +17,7 @@ internal static partial class ComponentIds
         /// <remarks>
         ///  As the name conflicts with <see cref="System.IServiceProvider"/> this is sometimes
         ///  called IOleServiceProvider, even though the name is actually IServiceProvider.
-        ///  <see cref="https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/cc678966(v=vs.85)"/>
+        ///  See https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/cc678966(v=vs.85)
```
Comment (sharwell) [inline src/System.Windows.Forms.Primitives/src/Interop/Ole32/Interop.IServiceProvider.cs]:
```
<see href="https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/platform-apis/cc678966(v=vs.85)">QueryService method (Microsoft Docs)</see>
```

File: src/System.Windows.Forms/src/System/Windows/Forms/KeyboardToolTipStateMachine.cs [hunk 3]
```diff
@@ -17,7 +17,7 @@ namespace System.Windows.Forms
     /// </summary>
     /// <remarks>
     ///  The control should be focused with keyboard for an amount of time specified with TTDT_INITIAL flag to make
-    ///  the keyboard ToolTip appear. <see cref="https://docs.microsoft.com/windows/win32/controls/ttm-getdelaytime"/>
+    ///  the keyboard ToolTip appear. See https://docs.microsoft.com/windows/win32/controls/ttm-getdelaytime
```
Comment (sharwell) [inline src/System.Windows.Forms/src/System/Windows/Forms/KeyboardToolTipStateMachine.cs]:
```
<see href="https://docs.microsoft.com/windows/win32/controls/ttm-getdelaytime">TTM_GETDELAYTIME message (Microsoft Docs)</see>
```

File: src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionUIService.cs [hunk 4]
```diff
@@ -60,7 +60,7 @@ public void ShowUI(IComponent component)
         }
 
         /// <summary>
-        ///  Refreshes the <see cref="DesignerActionGlyph">designer action glyph</see> as well as <see cref="DesignerActionPanel">designer action panels</see>.
+        ///  Refreshes the <see cref="System.Windows.Forms.Design.Behavior.DesignerActionGlyph">designer action glyph</see> as well as <see cref="DesignerActionPanel">designer action panels</see>.
```
Comment (sharwell) [inline src/System.Windows.Forms.Design/src/System/ComponentModel/Design/DesignerActionUIService.cs]: 💡 Using directive?

File: src/System.Windows.Forms.Primitives/src/Interop/IHandle.cs [hunk 5]
```diff
@@ -12,7 +12,7 @@
 ///  The key benefit of this is that we can keep the owning class from being
 ///  collected during interop calls. <see cref="HandleRef"/> wraps arbitrary
 ///  owners with target handles. Having this interface allows implicit use
-///  of the classes (such as <see cref="System.Windows.Forms.Control"/>) that
+///  of the classes (such as Control) that
```
Comment (sharwell) [inline src/System.Windows.Forms.Primitives/src/Interop/IHandle.cs]: 💡 This is a case where I would consider the full form to get a link in the final rendered documentation:

```
<see cref="T:System.Windows.Forms.Control"/>
```

File: src/System.Windows.Forms.Primitives/src/System/Windows/Forms/Message.cs [hunk 6]
```diff
@@ -21,29 +21,20 @@ public struct Message
...
-        /// <summary>
-        ///  Specifies the <see cref='Message.lparam'/> of the message.
-        /// </summary>
         public IntPtr LParam { get; set; }
 
         public IntPtr Result { get; set; }
 
         /// <summary>
-        ///  Gets the <see cref='Message.lparam'/> value, and converts the value to an object.
+        ///  Gets the <see cref='Message.LParam'/> value, and converts the value to an object.
```
Comment (sharwell) [inline src/System.Windows.Forms.Primitives/src/System/Windows/Forms/Message.cs]: 💭 Can this just be `cref="LParam"`?

File: src/System.Windows.Forms/src/System/Windows/Forms/DpiChangedEventHandler.cs [hunk 7]
```diff
@@ -5,9 +5,8 @@
 namespace System.Windows.Forms
 {
     /// <summary>
-    ///  Represents a method that will handle the <see cref='Control.DpiChanged'/>,
-    /// <see cref='Control.DpiChanged'/>, or <see cref='Control.DpiChanged'/>
-    ///  event of a <see cref='Control'/>.
+    ///  Represents a method that will handle the <see cref='Form.DpiChanged'/> event
+    ///  event of a <see cref='Form'/>.
```
Comment (sharwell) [inline src/System.Windows.Forms/src/System/Windows/Forms/DpiChangedEventHandler.cs]: 💡 The first line can end with `.` and the second line removed altogether.

File: src/System.Windows.Forms/src/System/Windows/Forms/TreeNodeMouseClickEventHandler.cs [hunk 8]
```diff
@@ -5,9 +5,8 @@
 namespace System.Windows.Forms
 {
     /// <summary>
-    ///  Represents a method that will handle the <see cref='TreeView.OnNodeClick'/>,
-    /// <see cref='TreeView.OnNodeClick'/>, <see cref='TreeView.OnNodeClick'/>,
-    ///  or <see cref='TreeView.BeforeSelect'/> event of a <see cref='TreeView'/>
+    ///  Represents a method that will handle the <see cref='TreeView.OnNodeMouseClick'/> or
+    /// <see cref='TreeView.OnNodeMouseDoubleClick'/> event of a <see cref='TreeView'/>.
```
Comment (sharwell) [inline src/System.Windows.Forms/src/System/Windows/Forms/TreeNodeMouseClickEventHandler.cs]: 💡 'of a TreeView' is unnecessary

File: src/System.Windows.Forms.Primitives/src/Interop/User32/Interop.SW.cs [hunk 9]
```diff
@@ -7,7 +7,8 @@ internal static partial class Interop
     internal static partial class User32
     {
         /// <summary>
-        ///  Show window flags for <see cref="ShowWindow"/> and <see cref="Shell32.ShellExecuteW"/>
+        ///  Show window flags for <see cref="ShowWindow(System.IntPtr, SW)"/> and
+        ///  <see cref="Shell32.ShellExecuteW(System.IntPtr, string, string, string, string, User32.SW)"/>
```
Comment (sharwell) [inline src/System.Windows.Forms.Primitives/src/Interop/User32/Interop.SW.cs]: 💡 Using directive for `System`

**Comment by RussKie** [comment]
Damn, I shouldn't have squashed...

**Pull request merged**